components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,749 @@
+= 0
+$$ (z-integration-mass)
+
+$$
+\frac{\partial }{\partial t} \int_{z_k^{bot}}^{z_k^{top}} \rho \varphi  dz
+ + \nabla_z \cdot \left( \int_{z_k^{bot}}^{z_k^{top}} \rho \varphi {\bf u} dz  \right)
+ + \int_{z_k^{bot}}^{z_k^{top}} \frac{\partial}{\partial z} \left(  \rho \varphi w \right) dz
+= \int_{z_k^{bot}}^{z_k^{top}} \left( D^\varphi + Q^\varphi \right) dz
+$$ (z-integration-tracers)
+
+This results in variables that are vertically mass-averaged, as defined in [](def-mass-thickness-average). The subscript $k$ denotes a variable that has been vertically mass-averaged over layer $k$. This system is now discrete in the vertical, but remains continuous in the horizontal and in time.
+
+$$
+\frac{\partial {\bf u}_k}{\partial t}
++ \zeta_k {\bf u}_k^{\perp}
++ f {\bf u}_k^{\perp}
++ \left[ \rho_k  w_k {\bf u}_k \right]^{top} - \left[ \rho_k w_k {\bf u}_k \right]^{bot}
+=
+- \frac{1}{\rho} \nabla_z p_k
+- \nabla_z K_k
++ {\bf D}_k^u + {\bf F}_k^u
+$$ (layered-momentum-1)
+
+$$
+\frac{\partial h_k}{\partial t} + \nabla_z \cdot \left(h_k {\bf u}_k\right) + \left[ \rho_k w_k \right]^{top} - \left[ \rho_k w_k \right]^{bot}= Q^h_k
+$$ (layered-mass-1)
+
+$$
+\frac{\partial h_k \varphi_k}{\partial t} + \nabla_z \cdot \left(h_k {\bf u}_k \varphi_k\right)
++ \left[ \varphi_k \rho_k w_k \right]^{top} - \left[ \varphi_k \rho_k w_k \right]^{bot}
+=  D^\varphi_k + Q^\varphi_k.
+$$ (layered-tracer-1)
+
+
+Equivalently, one could have multiplied  [](z-integration-momentum), [](z-integration-mass), [](z-integration-tracers) by $g$ and expressed the integrals in terms of pressure, with integration limits from $p_k^{top}$ to $p_k^{bot}$.
+Derivations of ocean model equations with pressure as the vertical variable may be found in [de Szoeke and Samelson 2002](https://journals.ametsoc.org/view/journals/phoc/32/7/1520-0485_2002_032_2194_tdbtba_2.0.co_2.xml) and [Losch et al. 2003](https://journals.ametsoc.org/view/journals/phoc/34/1/1520-0485_2004_034_0306_hsacgc_2.0.co_2.xml). 
+
+The vertical advection terms use the fundamental theorem of calculus to integrate a derivative in $z$, resulting in boundary conditions at the layer interfaces. This makes intuitive sense as a mass balance. For example, in the absence of horizontal advection and sources, the mass equation is simply
+
+$$
+\frac{\partial h_k}{\partial t}
+= \left[ \rho_k w_k \right]^{bot} - \left[ \rho_k w_k \right]^{top}.
+$$ (layered-mass-vert-adv)
+
+This states that the change in mass in the layer is the incoming mass from below minus the outgoing mass above, since the vertical velocity $w$ is positive upwards. The vertical advection of momentum and tracers have a similar interpretation.
+
+### Questions
+
+1. How do integrals in  [](#z-integration-momentum) to [](#z-integration-tracers) turn into the layered form [](layered-momentum-1) to [](layered-tracer-1) when there are products in the terms? For example, $\zeta_k {\bf u}^\perp_k$, $\nabla K$ and $\rho \varphi {\bf u}$.
```

Comment:
I believe MOM6 (and perhaps other models as well) assume that prognostic variables ($\Theta, S_A,  {\bf u}^\perp$, etc.) are vertically constant in a layer and have jumps at layer interfaces.  This allows for a mathematically consistent treatment.  I believe this assumption is not require within layers (e.g. for the pressure gradient, they allow $\Theta$ and $S_A$ to vary linearly between neighboring cells in the same layer).

Reply:
Thanks, that is helpful. I will update the document but wanted to think about that further.

Reply:
Just seconding what Xylar said, T,S, and the specific volume are considered constant (vertically) within a layer. P considered linear in the vertical. T,S considered linear in the horizontal